akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -456,6 +532,12 @@ final class ClusterClientReceptionist(system: ExtendedActorSystem) extends Exten
         .withDispatcher(dispatcher), name)
     }
   }
+
+  /**
+   * Returns the underlying receptionist actor.
+   */
+  val underlying: ActorRef =
+    receptionist
```

Comment:
No, it is generally useful so that you can filter the events received by the receptionist you're interested in. I shall beef up the comment.